api/models/formation.go

```diff
@@ -51,9 +52,19 @@ func ListFormation(app string) (Formation, error) {
 	formation := Formation{}
 
 	for _, me := range manifest {
-		count, _ := strconv.Atoi(a.Parameters[fmt.Sprintf("%sDesiredCount", UpperName(me.Name))])
-		memory, _ := strconv.Atoi(a.Parameters[fmt.Sprintf("%sMemory", UpperName(me.Name))])
-		cpu, _ := strconv.Atoi(a.Parameters[fmt.Sprintf("%sCpu", UpperName(me.Name))])
+		var count, memory, cpu int
+
+		if vals, ok := a.Parameters[fmt.Sprintf("%sFormation", UpperName(me.Name))]; ok {
+			parts := strings.SplitN(vals, ",", 3)
```

Comment:
Good call. One more pass to review validation and to add some tests certainly wouldn't hurt.